storm-core/src/jvm/org/apache/storm/metric/FileBasedEventLogger.java

```diff
@@ -45,7 +46,7 @@
 
     private void initLogWriter(Path logFilePath) {
         try {
-            LOG.info("logFilePath {}", logFilePath);
+            LOG.info("FileBasedEventLogger log path {}", logFilePath);
```

Comment:
Minor nit: we probably don't need the classname repeated here, since it is part of the Logger configuration.